### ISSUE TITLE
<para> Note appears 4 times

### ISSUE THREAD

**Issue by jh-isw** [aas-core-works/aas-core3.0-csharp#16]
see this line

https://github.com/aas-core-works/aas-core3.0-csharp/blob/99870fd472c2ff42262d281546f53895bbffc8b7/src/AasCore.Aas3_0/types.cs#L10173

and the following

**Comment by mristin** [comment]
@jh-isw thanks for having a look! I thought that was the intended way to mark the different paragraphs?

**Comment by jh-isw** [comment]
I meant it's 4 times more or less the same content:

```
    /// <para>
    /// Note: categories are deprecated since V3.0 of Part 1a of the document series
    /// "Details of the Asset Administration Shell".
    /// </para>
    /// <para>
    /// Note: categories are deprecated since V3.0 of Part 1a of the document series
    /// "Details of the Asset Administration Shell".
    /// </para>
    /// <para>
    /// Categories are deprecated since V3.0 of Part 1a of the document series
    /// "Details of the Asset Administration Shell".
    /// </para>
    /// <para>
    /// Categories are deprecated since V3.0 of Part 1a of the document series
    /// "Details of the Asset Administration Shell".
    /// </para>
```

**Comment by mristin** [comment]
Thanks! This is an issue with indention in the docstring of aas-core-meta: https://github.com/aas-core-works/aas-core-meta/blob/2a81612dab519215621c55c8e3b434f441e5f588/aas_core_meta/v3.py#L4375. The notes should be indented one level more.

I'll try to fix this tomorrow.

**Comment by mristin** [comment]
Fixed by pull request #18. Thanks again for reporting the issue!
